Gemfile

```diff
@@ -6,3 +6,4 @@ gem 'slather',
   :branch   => "feature-profdata"
 gem 'xcpretty'
 gem 'xcpretty-travis-formatter'
+gem 'carthage'
```

Comment:
You need to check in any changes to a gemfile with the gemfile.lock. I'd just install this gem afterwards in `deploy.sh` if all goes well prior to that step.